mmv1/third_party/terraform/services/eventarc/resource_eventarc_enrollment_test.go

```diff
@@ -0,0 +1,51 @@
+package eventarc_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func testAccCheckEventarcEnrollmentDestroyProducer(t *testing.T) func(s *terraform.State) error {
```

Comment:
What's the reason for adding this destroy function here?

Reply:
This is called in the Enrollment tests I added to resource_eventarc_message_bus_test.go: https://github.com/GoogleCloudPlatform/magic-modules/pull/13284/files/1cf54ce5e52fbc78b13638c343664ecf9de9da30#diff-2d6ada4881485985c28664e3209e49efcfecd4b5f2c11db1b8bab6f750a5f928R425. This should check that the Enrollment was properly destroyed by the end of the test?

Reply:
I just wonder if it's necessary to put this function in a new file, referring to `testAccCheckEventarcMessageBusDestroyProducer`.

Reply:
It's not necessary. I'll remove it